test/cpp/interop/backend_metrics_lb_policy.h

```diff
@@ -29,23 +29,19 @@
 namespace grpc {
 namespace testing {
 
-constexpr absl::string_view kMetricsTrackerArgument = "orca_metrics_tracker";
+using xds::data::orca::v3::OrcaLoadReport;
 
 class LoadReportTracker {
  public:
-  std::map<std::string, void*> channel_arguments() {
-    return {{std::string(kMetricsTrackerArgument), this}};
-  }
-
+  ChannelArguments GetChannelArguments();
+  void ResetCollectedLoadReports();
   void RecordPerRpcLoadReport(
       const grpc_core::BackendMetricData* backend_metric_data);
-
-  std::deque<absl::optional<xds::data::orca::v3::OrcaLoadReport>>
-  GetCollectedPerRpcLoadReports();
+  absl::StatusOr<absl::optional<OrcaLoadReport>> GetFirstLoadReport();
```

Comment:
I don't think we need a `StatusOr<>` here, just an `optional<>`.  I realize that makes two levels of optionals, which is a little awkward, but I think you can make it clearer with a typedef:

```
// A load report, or nullopt if the call had no load report.
using LoadReportEntry = absl::optional<xds::data::orca::v3::OrcaLoadReport>;

// Returns the next load report, or nullopt if the queue is empty.
absl::optional<LoadReportEntry> GetNextLoadReport();
```